op-supervisor/supervisor/backend/backend.go

```diff
@@ -488,15 +582,28 @@ func (su *SupervisorBackend) CheckAccessList(ctx context.Context, inboxEntries [
 		}
 		entries = remaining
 
-		msgBlock, err := su.checkAccess(acc, executingDescriptor)
+		// Check if message passes time checks
+		if err := executingDescriptor.AccessCheck(su.depSet.MessageExpiryWindow(), acc.Timestamp); err != nil {
+			return err
+		}
+
+		var msgBlockDB eth.BlockID
+		if rpcVerifyAccess {
+			msgBlockDB, err = su.verifyAccessWithRPCInParallel(ctx, executingDescriptor, acc, func(acc types.Access) (eth.BlockID, error) {
+				return su.checkAccessWithDB(acc, executingDescriptor)
+			})
+		} else {
+			msgBlockDB, err = su.checkAccessWithDB(acc, executingDescriptor)
+		}
 		if err != nil {
 			su.logger.Debug("Access-list inclusion check failed", "err", err)
 			return types.ErrConflict
```

Comment:
If there is an inconsistency between db and remote RPC, it will throw `ErrConflict` which may cause safe reorgs. Is this the intended behavior? Maybe we need to define what happens during inconsistency detected. I think we also need to distinguish between RPC errors and the actual inconsistency.

Reply:
Makes sense. The RPC verification now is purely for alerting purposes, see that the error is swallowed [here](https://github.com/ethereum-optimism/optimism/blob/1f8ec57b9c2d7951a1e69077ee69437256ba764b/op-supervisor/supervisor/backend/backend.go#L455-L460).